index.bs

```diff
@@ -87,9 +87,17 @@ The side effect of running the logger function with data and logLevel set to err
 
 The side effect of running the logger function with data and logLevel set to info.
 
-<h4><dfn method for="Console">log()</dfn></h4>
+<h4><dfn method for="Console">log([arg1], [arg2], ...)</dfn></h4>
 
-The side effect of running the logger function with data and logLevel set to log.
+Prints to the console as side effect of running the logger function if the logLevel is set to log. If the log function is called with no arguments, no output is written to the console. Each log entry from console.log gets an own line in the console. It is possible to pass format specifiers to the function in the first argument.
```

Comment:
Should this be "and the logLevel is set to log."?

Would this algorithm be better with the logger function?